RELEASE.md

```diff
@@ -20,11 +20,9 @@ Releases should be twice monthly<sup>[1](#footnote1)</sup> on the second and for
 
 ### Kubernetes
 
-The Addons in this repo are the minimum base set of supported<sup>[2](#footnote2)</sup> Addons to be installed as a suite for any supported version of Kubernetes.
+**The Kubernetes versions supported by this repo are 1.15, 1.16, and 1.17.**
```

Comment:
Wouldn't we need a 1.17 release to support Kubernetes 1.17? I don't see one in the releases of this repo and I see that https://jira.d2iq.com/browse/D2IQ-68046 is integrating without seeing a related PR. Am I missing something? I ask this following a question from a customer: https://mesosphere.slack.com/archives/CUPQBG3K9/p1591107332196000

Reply:
A 1.17 release hasn't happened yet but is currently in testing (1.9.0): https://github.com/mesosphere/kubernetes-base-addons/commits/testing-1.9.0

This is what we're using to test MWT on a 1.17 cluster so the next KBA release will include a `stable-1.17-xx`

Reply:
nvm! looks like @branden had been testing 1.8.0 on a 1.17 cluster and he just released https://github.com/mesosphere/kubernetes-base-addons/releases/tag/stable-1.17-1.8.0

Reply:
Great, thanks for the info!